articles/role-based-access.md

```diff
@@ -64,7 +64,9 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 | Filter hosts using policies                                                                                                                | ✅       | ✅         | ✅         | ✅    |         |
 | Create, edit, and delete policies for all hosts                                                                                            |          |            | ✅         | ✅    | ✅      |
 | Create, edit, and delete policies for all hosts assigned to team\*                                                                         |          |            | ✅         | ✅    | ✅      |
-| Manage [policy automations](https://fleetdm.com/docs/using-fleet/automations#policy-automations)                                           |          |            |            | ✅    | ✅      |
+| Edit global ("All teams") policy automations                  |          |            |            | ✅    | ✅      |
+| Edit team policy automations: calendar events, install software, and run script\* |          |            | ✅         | ✅    | ✅      |
+| Edit team policy automations: other workflows (tickets and webhooks)\*                 |          |            |            | ✅    | ✅      |
```

Comment:
Confirming we want to drop docs links here? Seemed liked a good opportunity to show off functionality by linking out to guides for each feature.

Reply:
Yeah this was intentional. Make things easier for our future selves.

Currently, the guides for policy automations are sprawled out over 3 guides:
- https://fleetdm.com/guides/automations
- https://fleetdm.com/guides/policy-automation-run-script
- https://fleetdm.com/guides/automatic-software-install-in-fleet

We might combine these guides in the future. If we have links to one of these guides in the permissions guide here we have to remember to update it.

Reply:
Hmm, okay. I figured that if we combined guides we would throw in redirects from the old URLs to avoid dead links (which is why the existing automation guide link worked), but fair enough.